README.md

```diff
@@ -31,6 +31,37 @@ module.exports = {
 };
 ```
 
+### Utility Classes
+
+The simplest way to apply Garden Design Tokens is by using the
+[Tailwind utility classes](https://tailwindcss.com/docs/utility-first).
+
+```html
+<img
+  class="rounded-full h-8 w-8 border border-green-600"
+  src="avatar.png"
+  alt="Avatar" />
+```
+
+To reduce the bundle size impact of these utilities consider using the [PurgeCSS functionality](https://tailwindcss.com/docs/controlling-file-size#removing-unused-css)
+provided within Tailwind.
+
+### Class Composition with `@apply`
+
+Tailwind provides several [PostCSS directives](https://tailwindcss.com/docs/functions-and-directives/)
+to consume utility styles. The [@apply directive](https://tailwindcss.com/docs/functions-and-directives/#apply)
```

Comment:
"...~to consume~ for inserting utility styles into CSS."

Reply:
Updated.